js/packages/teams-ai/src/authentication/Authentication.ts

```diff
@@ -339,6 +339,11 @@ export type OAuthSettings = OAuthPromptSettings & {
      * Optional. Set this to enable SSO when authentication user using adaptive cards.
      */
     tokenExchangeUri?: string;
+
+    /**
+     * Optional. Set to `true` to enable SSO when authenticating using AAD.
```

Comment:
Should we add migration documentation for this feature?

Reply:
Thanks for calling this out I've added a new section to the `USER-AUTH.md` concept docs